core/src/main/resources/application.yml

```diff
@@ -16,7 +16,8 @@ logging:
 ethereum:
   nodes:
     - name: default
-      url: https://baklava-forno.celo-testnet.org
+      #url: https://baklava-forno.celo-testnet.org
```

Comment:
Maybe we should let localhost:8545 by default, before we refactor the way of get the application.ymal

Reply:
fixed